### PR TITLE
Fix CLI docstring and iPython version specifier

### PR DESCRIPTION
**Pull request by janja-janja**

### CODE REVIEW

File: fummy.py [hunk 1]
```diff
@@ -11,11 +11,14 @@
     the main branch
 """
 import os
-import datetime
```
Comment (janja-janja) [inline fummy.py:14]: This

``` python
import datetime 
```

causes some fun and games. See [StackOverflow answer](http://stackoverflow.com/questions/12906402/type-object-datetime-datetime-has-no-attribute-datetime).

``` python
from datetime import datetime 
```

The above approach is always recommended

Comment (ghost) [inline fummy.py:14]: Haha... All games I have played. But yeah i'll make that edit.

Comment (janja-janja) [inline fummy.py:14]: hehe check PR #3

File: README.md [hunk 2]
```diff
@@ -17,6 +17,6 @@ and merging branches to master you leave a lot of unused branches in your reposi
 
 Example:
 
-``` python
+``` bash
```
Comment (ikosenn) [inline README.md:20]: why this change?

File: fummy.py [hunk 3]
```diff
@@ -85,8 +88,9 @@ def kill_merged(default):
 @click.group()
 def cli():
     """
-    Development helpers for the EMR project
-    These utilities help with testing, loading of data, database resets etc
+    Command Line Interface tools loader for  ``fummy``
+    These utilities help with deleting git branches older than the specified
+    period
```
Comment (ikosenn) [inline fummy.py:93]: haha..... Hiyo nilicargo cult

File: README.md (hunk 2 above)
Comment (dkaranja) [inline README.md:20]: Im thinking it's a bash command since we're not running it inside the py interpreter

File: fummy.py (hunk 3 above)
Comment (dkaranja) [inline fummy.py:93]: haha kama master
